tools/amd_build/build_amd.py

```diff
@@ -193,6 +193,7 @@ def remove_hcc(line: str) -> str:
     output_directory=out_dir,
     includes=includes,
     ignores=ignores,
+    extra_files=["torch/_inductor/codegen/wrapper.py"],
```

Comment:
All internal builds are failing due to this line.

Reply:
Ok, found a workaround by providing a dummy to fb_caffe2_hipify.sh, but need to think of a proper fix. cc: @xw285cornell @bertmaher